src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/PathGuidanceType.java

```diff
@@ -118,6 +119,17 @@ public static GraphQLObjectType create() {
           )
           .build()
       )
+      .field(
+        GraphQLFieldDefinition
+          .newFieldDefinition()
+          .name("elevationProfile")
+          .description("The step's elevation profile.")
+          .type(new GraphQLNonNull(new GraphQLList(elevationStepType)))
+          .dataFetcher(environment ->
+            ((WalkStep) environment.getSource()).getElevationProfile().stepsWithoutUnknowns()
```

Comment:
The data-fetcher here is different from the LegType, could the same mapping be used in both places?